src/images/wavedrom/reg-based-ldnstr.adoc

```diff
@@ -5,9 +5,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op', attr: ['2', 'C0', 'C0', 'C0', 'C0', 'C0'], type: 8},
-  {bits: 3, name: 'rd`',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
+  {bits: 3, name: 'rd',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
   {bits: 2, name: 'imm', attr:['2', 'offset[2|6]', 'offset[7:6]', 'offset[7:6]', 'offset[2|6]', 'offset[7:6]'], type: 2},
-  {bits: 3, name: 'rs1`', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},
+  {bits: 3, name: 'rs1', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},
```

Comment:
The prime should be retained.. ` should have been changed to '

Reply:
Ah so it looks like `{bits: 3, name: 'rs1'', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},` ?

Is this true for all of them?

Reply:
Yes, all instances where `rs1` or `rs2` or `rd` is a 3-bit field then its marked with ' to indicate that it holds the restricted set of registers (as compared to when they are 5-bit fields..

Reply:
The #1335 reported four typos where ` was used instead of '.